modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -107,4 +105,82 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-180358. Verify all status changes after 'need more info' request"
+	@priority = 5
+	test VerifyStatusAfterRequestMoreInfo {
+		task ("Given that a new organization was created with user employee") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "employee@liferay.com",
+				userLoginFullName = "Employee User");
+
+			EVPUtils.gotoEVP();
+
+			EVPOrganization.goToCreateOrganization(
+				button = "Create an Organization",
+				navItem = "Organization");
+
+			EVPOrganization.fillOrganizationForm(
+				bankingInfo = "Banking Info",
+				city = "City",
+				contactEmail = "Contact Email",
+				contactName = "Contact Name",
+				contactPhone = "Contact Phone",
+				country = "Country",
+				done = "true",
+				organizationName = "Organization 5",
+				organizationSite = "Social Media Link",
+				smallDescription = "Small Description",
+				state = "State",
+				street = "Street",
+				taxId = "Tax Id",
+				zipCode = "Zip");
+		}
+
+		task ("When the Manager click in 'Need more info'") {
```

Comment:
this step action should be  When the Manager requests more info.

When the Manager click in 'Need more info shows only one action.

Reply:
The flow of my test is different from the macro it is creating, I took this doubt with it.